test/api/controllers/memoryStorageController.test.js

```diff
@@ -430,20 +428,20 @@ describe('Test: memoryStore controller', function () {
 
     it('should throw an error an invalid keys parameter is given', () => {
       var
-        requestObject = new RequestObject({
+        request = new RequestObject({
           body: {
             keys: 'unvalid value'
           }
         });
 
-      should(extractArgumentsFromRequestObjectForZInterstore.bind(null, requestObject)).throw(BadRequestError);
+      should(extractArgumentsFromRequestObjectForZInterstore.bind(null, request)).throw(BadRequestError);
 
-      requestObject = new RequestObject({
+      request = new RequestObject({
         body: {
           weights: 'unvalid'
         }
       });
-      should(extractArgumentsFromRequestObjectForZInterstore.bind(null, requestObject)).throw(BadRequestError);
+      should(extractArgumentsFromRequestObjectForZInterstore.bind(null, request)).throw(BadRequestError);
 
     });
   });
```

Comment:
Just curious: Is there a reason to rename all requestObject variables to request?